config/module.go

```diff
@@ -258,8 +257,13 @@ func (m *Module) FirstRun(
 		return err
 	}
 
-	// check if FirstRun already ran successfully for this module version by
-	// checking if a success marker file exists on disk.
+	// check if FirstRun already ran successfully for this module version by checking if a success
+	// marker file exists on disk. An example module directory structure:
+	//
+	// .viam/packages/data/module/e76d1b3b-0468-4efd-bb7f-fb1d2b352fcb-viamrtsp-0_1_0-linux-amd64/
+	// .viam/packages/data/module/e76d1b3b-0468-4efd-bb7f-fb1d2b352fcb-viamrtsp-0_1_0-linux-amd64/bin/
+	// .viam/packages/data/module/e76d1b3b-0468-4efd-bb7f-fb1d2b352fcb-viamrtsp-0_1_0-linux-amd64/bin.first_run_succeeded
```

Comment:
I don't think this was intentional. Or maybe it was. But it's certainly a bit odd to concatenate a "directory" with a "file suffix" without some "file prefix".

I started to "fix" this by using `filepath.Join`. And additionally being "backwards compatible" by checking for both `bin.first_run_succeeded` as well as `bin/.first_run_succeeded`.

In the end I figured it's not worth risking breaking this code for aesthetics. But found it useful to document a concrete example such that readers aren't left wondering if something is wrong.